masu/database/sql/reporting_ocpusagelineitem_aggregates.sql

```diff
@@ -7,10 +7,10 @@ CREATE TEMPORARY TABLE reporting_ocpusagelineitem_aggregates_{uuid} AS (
         li.node,
         sum(li.pod_usage_cpu_core_seconds) / 3600 as pod_usage_cpu_core_hours,
         sum(li.pod_request_cpu_core_seconds) / 3600 as pod_request_cpu_core_hours,
-        max(li.pod_limit_cpu_cores) as pod_limit_cpu_cores,
+        max(li.pod_limit_cpu_core_seconds) as pod_limit_cpu_core_hours,
```

Comment:
We can probably get rid of the aggregates file for now, but this has seconds being called hours without unit conversion